### DISCUSSION TITLE
Set Additional Print Columns with JOSDK

### DISCUSSION THREAD

**Discussion by coltmcnealy-lh** [operator-framework/java-operator-sdk#1973] [Q&A] [answered]
Does the JOSDK support the ability to set additional print columns on a `Foo` CRD when I do `kubectl get foo`?

**Comment by metacosm** [comment, marked as answer]
This question is more appropriate for the Fabric8 client project but yes, you should be able to do so, see: https://github.com/fabric8io/kubernetes-client/blob/009974fe5fe634cbd6d1210780b99ff82e251f2d/crd-generator/api/src/test/java/io/fabric8/crd/example/joke/JokeRequestSpec.java#L42
More generally speaking, you can see some of the features of the CRD generator here: https://github.com/fabric8io/kubernetes-client/blob/master/doc/CRD-generator.md, though, ironically enough, dealing with printer columns is not documented there 😞
